src/auth/AuthContext.tsx

```diff
@@ -136,7 +137,7 @@ export default function AuthContextProvider({children}: PropsWithChildren<{}>) {
         ) {
           return 'invalid_code';
         }
-        console.warn(error);
+        if (isAuthError(error)) Bugsnag.notify(error);
```

Comment:
Wait!, I approved it, but now I saw something, hehe. 

```if (isAuthError(error)) Bugsnag.notify(error);``` Should go first before `if (isAuthError(error) && error.code === ERROR_INVALID_PHONE_NUMBER)` otherwise that case will be ignored and not logged.

Reply:
We might not be interested in knowing whenever someone types the wrong code. What we need for this issue is to know what kind of error type we get when it is not an invalid code, but the user still cannot log in